### PR TITLE
Using pnpm 7.3, resolve strict peer errors

### PR DESCRIPTION
**Pull request by NullVoxPopuli**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -43,7 +43,7 @@
     "remove-types": "^1.0.0",
     "serve": "^13.0.2",
     "typedoc": "^0.23.2",
-    "typescript": "4.8.0-dev.20220627",
```
Comment (NullVoxPopuli) [inline package.json]: typedoc declares typescript as a peer, and does not include `dev` releases

Comment (wycats) [reply package.json]: Is this actually ok?

Comment (wycats) [reply package.json]: Can we just tell pnpm to ignore the error?

Comment (NullVoxPopuli) [reply package.json]: that's an option, yeah -- I'll switch to that since we're using 4.8-dev everywhere

**Pull request merged**